src/pages/workspaces/workspace/Tools.js

```diff
@@ -18,22 +20,63 @@ const styles = {
     padding: '1rem 4rem',
     display: 'flex', flexWrap: 'wrap'
   },
-  card: {
+  shortCard: {
     ...Style.elements.card, width: 300, height: 125, margin: '1rem 0.5rem',
     display: 'flex', flexDirection: 'column', justifyContent: 'space-between'
   },
   shortTitle: {
-    flex: 'none',
+    flex: 1,
     color: colors.blue[0], fontSize: 16,
     lineHeight: '20px', height: '40px',
     overflow: 'hidden', overflowWrap: 'break-word'
-  }
+  },
+  shortDescription: {
+    flex: 'none',
+    lineHeight: '18px', height: '90px',
+    overflow: 'hidden'
+  },
+  longCard: {
+    ...Style.elements.card,
+    width: '100%', minWidth: 0,
+    margin: '0.25rem 0.5rem'
+  },
+  longTitle: {
+    color: colors.blue[0], fontSize: 16,
+    whiteSpace: 'nowrap', overflow: 'hidden', textOverflow: 'ellipsis', flex: 1
+  },
+  longDescription: {
+    flex: 1,
+    paddingRight: '1rem',
+    whiteSpace: 'nowrap', overflow: 'hidden', textOverflow: 'ellipsis'
+  },
+  toolbarContainer: {
+    flex: 'none', display: 'flex', alignItems: 'flex-end',
+    margin: '1rem 4.5rem'
+  },
+  toolbarButtons: {
+    marginLeft: 'auto', display: 'flex',
+    backgroundColor: 'white', borderRadius: 3
+  },
+  toolbarButton: active => ({
+    display: 'flex', justifyContent: 'center', alignItems: 'center',
+    height: '2.25rem', width: '3rem',
+    color: active ? colors.blue[1] : colors.blue[0]
+  })
 }
 
-const ToolCard = pure(({ name, namespace, config }) => {
+const ToolCard = pure(({ listView, name, namespace, config }) => {
   const { namespace: workflowNamespace, name: workflowName, methodRepoMethod: { sourceRepo, methodVersion } } = config
-  return a({
-    style: styles.card,
+  return listView ? a({
+    style: styles.longCard,
+    href: Nav.getLink('workflow', { namespace, name, workflowNamespace, workflowName })
+  }, [
+    div({ style: { display: 'flex', alignItems: 'center' } }, [
+      div({ style: styles.longTitle }, [workflowName]),
+      div({ style: { marginRight: '1rem' } }, [`V. ${methodVersion}`]),
+      div({ style: { flex: 'none' } }, [`Source: ${sourceRepo}`])
```

Comment:
nit: If the source text lengths are different, these won't align across rows. I know we're deviating from the original design a bit because we don't have the description field available, but I suspect that design are thinking of this like a 'table' mode, and would prefer to have things line up if possible.